target/arc/cpu.c

```diff
@@ -191,6 +191,25 @@ static void arc_cpu_reset(DeviceState *dev)
      * DSP=0x1: MPY_OPTION 7
      */
     cpu->mpy_build = 0x00001006;
+
+    /*
+     * Some runtime libraries check build registers to
+     * determine enabled features. Thus, it's necessary
+     * to configure them properly.
+     */
+    switch (cpu->family) {
+    case ARC_OPCODE_ARC600:
```

Comment:
QEMU port for ARC doesn't support ARCompact processors, i.e. ARC600 & ARC700 families. So most likely we may drop these 2 cases.